spec/registry.md

```diff
@@ -619,7 +619,17 @@ All other values produce an _Invalid Expression_ error.
 > or the type `com.ibm.icu.util.CurrencyAmount` can be used to set the currency and related
 > options (such as the number of fraction digits).
 
+### Non-Negative Integer Options
 
+Some _options_ of number _functions_ are defined to take a "non-negative integer".
+In most cases, these control the number of various kinds of digits.
+
+A "non-negative integer" is one of:
+- an implementation-defined integer numeric type where the value of the 
+  encoded number is not less than zero
+- a literal consisting only of ASCII digits U+0030 through U+0039;
```

Comment:
Saying "literal" here would mean that a variable reference resolving to a numeric string value would not be valid. This is problematic because it would introduce a new requirement for a function implementation to be able to tell apart a literal string value and a variable string value.

```suggestion
- a string consisting only of ASCII digits U+0030 through U+0039;
```

Reply:
I said "literal" to allow the value to be quoted, e.g. `maximumFractionDigits=|11|`

I agree that my phrasing wouldn't allow a variable reference, but the item to fix to accomplish that is the first one. Change to follow...